tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java

```diff
@@ -24,7 +25,11 @@ public TesAdapterController(TesTaskService tesTaskService) {
     }
 
     @GetMapping("/v1/tasks")
-    public ResponseEntity<TesListTasksResponse> listTesTasks() {
-        return new ResponseEntity<>(tesTaskService.listTesTask(), HttpStatus.OK);
+    public ResponseEntity<TesListTasksResponse> listTesTasks(@RequestParam("name_prefix") String name_prefix,
+                                                             @RequestParam("page_size") int page_size,
+                                                             @RequestParam("page_token") String page_token,
+                                                             @RequestParam("view") Enum view,
```

Comment:
We need one param with `view` with default value `MINIMAL`.
1. Add `TaskView` enum in model package with values `MINIMAL, BASIC, FULL`
2. Use `TaskView` as parameter type
3. Specify default value `@RequestParam(....., defaultValue = "MINIMAL")`